lib/internals/routeHandler.js

```diff
@@ -74,21 +84,20 @@ class RouteHandler {
                 .bind(this)
                 .then(() => {
 
```

Comment:
Can I convince you to remove this `Promise` chain and just return the contents of the last `then`able in the chain?